data/html-generator.js

```diff
@@ -0,0 +1,19 @@
+const fs = require('fs');
```

Comment:
This is the script that generates the HTML files from the react JS classes for each district / state.